content/about/introduction/introduction.html

```diff
@@ -14,166 +14,161 @@
         </p>
         <ul>
-          <li>Are consistent with the intent of ARIA, HTML, CSS, and related specifications.</li>
-          <li>Support assistive technology interoperability, which is assessed by the <a href="https://aria-at.w3.org/">ARIA and Assistive Technologies Project</a>.</li>
-          <li>Promote practical techniques for developing modern GUIs that both support usability for people with disabilities and conform with <a href="https://www.w3.org/TR/WCAG21/">WCAG</a> requirements.</li>
+          <li>ARIA、HTML、CSS、および関連する仕様の意図と一致している。</li>
+          <li>支援技術の相互運用性をサポートし、<a href="https://aria-at.w3.org/">ARIA and Assistive Technologies Project</a>によって評価される。</li>
+          <li>障害を持つ人々の使用可能性をサポートし、<a href="https://www.w3.org/TR/WCAG21/">WCAG</a>の要件に準拠する、現代のGUIを開発するための実用的な技術を推進する。</li>
         </ul>
       </section>
 
       <section id="apg-is--not-normative">
-        <h2>APG is Not a Normative Standard</h2>
+        <h2>APGは規範的な標準ではありません</h2>
         <p>
-          The accessibility guidance in the APG is different from accessibility requirements specified by <a href="https://www.w3.org/TR/WCAG21/">WCAG</a> and <a class="specref" href="#abstract">ARIA</a>.
-          WCAG and ARIA are "W3C normative technical standards" while the APG is an "informative" resource.
-          W3C normative technical standards, which are formally known as "W3C Recommendations", have conformance models that define how to comply with the requirements documented in the standard.
-          Those requirements are developed using a rigorous process defined by
-          The <a href="https://www.w3.org/Consortium/Process#rec-track">W3C Recommendation Track</a>.
+          APGのアクセシビリティガイダンスは、<a href="https://www.w3.org/TR/WCAG21/">WCAG</a>および<a class="specref" href="#abstract">ARIA</a>によって指定されたアクセシビリティ要件とは異なります。
+          WCAGとARIAは「W3C規範的技術標準」であり、APGは「情報提供」リソースです。
+          「W3Cの勧告」と公式に知られているW3C規範的技術標準には、標準で文書化された要件に準拠する方法を定義する適合モデルがあります。
+          これらの要件は、<a href="https://www.w3.org/Consortium/Process#rec-track">W3C Recommendation Track</a>によって定義された厳格なプロセスを使用して開発されます。
         </p>
         <p>
-          The APG does not specify normative requirements and thus does not have a conformance model.
-          The <a href="https://www.w3.org/groups/tf/aria-practices">ARIA Authoring Practices Task Force</a>, which is part of the <a href="https://www.w3.org/groups/wg/aria">ARIA Working Group</a>,
-          uses consensus to ensure APG guidance is consistent with all relevant specifications, including ARIA, but its process for doing so is not bound by the W3C Recommendation Track process.
+          APGは規範的な要件を指定せず、したがって適合モデルを持っていません。
+          <a href="https://www.w3.org/groups/tf/aria-practices">ARIA Authoring Practices Task Force</a>は、<a href="https://www.w3.org/groups/wg/aria">ARIA Working Group</a>の一部であり、APGガイダンスがARIAを含むすべての関連仕様と一致していることを確実にするために合意を使用しますが、そのプロセスはW3Cの勧告トラックプロセスによって束縛されません。
```

Comment:
中ほどの文節がややわかりにくいかなと思ったので以下の通りご提案してみます。

```suggestion
          <a href="https://www.w3.org/groups/tf/aria-practices">ARIA Authoring Practices Task Force</a>は、<a href="https://www.w3.org/groups/wg/aria">ARIA Working Group</a>の一部であり、APGのガイダンスがARIAを含むすべての関連仕様と一致することに合意していますが、そのプロセスはW3Cの勧告トラックプロセスによって束縛されません。
```